webpack/scenes/ContentViews/Copy/CopyContentViewModal.js

```diff
@@ -1,17 +1,21 @@
+/* eslint-disable react/no-danger */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Modal, ModalVariant } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
 import CopyContentViewForm from './CopyContentViewForm';
 
 const CopyContentViewModal = ({
   cvId, cvName, show, setIsOpen,
 }) => {
   const description = (
-    <p>
-      This will create a copy of <b>{cvName}</b>, including details,
-      repositories, and filters. Generated data such
-      as history, tasks and versions will not be copied.
-    </p>
+    <p
+      dangerouslySetInnerHTML={{
+        __html: __(`This will create a copy of <b>${cvName}</b>, including details,
```

Comment:
`dangerouslySetInnerHTML` is bad, no?  Too bad we don't have any other way to translate text with formatting. :thinking:

Reply:
yeah, we should investigate more about it,
though for now, I think it's the only solution...